bedrock/firefox/templates/firefox/features/safebrowser.html

```diff
@@ -0,0 +1,222 @@
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import call_out_compact, hero with context %}
+
+{% extends "firefox/base-protocol.html" %}
+
+{% block page_title %}{{ _('A safe browser that fights for you') }}{% endblock %}
+
+{% block page_desc %}
+{{ _('You have the right to internet safety. That’s why Firefox has features and add-ons that help us make the web more
+secure.') }}
+{% endblock %}
+
+{% block page_css %}
+{{ css_bundle('safebrowser') }}
+{% endblock %}
+
+{% block content %}
+<div class="mzp-c-hero">
+  <div class="hero-content">
+    <img src="{{ static('protocol/img/logos/firefox/firefox.png') }}" width="50" height="50" alt="">
+    <p>{{ _('Internet Security') }}</p>
+    <h1>{{ _('Take the stress out of finding a safe browser.') }}</h1>
+    <div class="download-firefox">
+      {{ download_firefox(dom_id='safebrowser-hero-download', download_location='primary cta') }}
+    </div>
+  </div>
+</div>
+<div class="mzp-l-content">
+  <article class="mzp-c-article">
+    <p class="mzp-c-article-intro">
+      {% trans %}
+      Building a safe browser in an art and a science because there are no set rules. At Firefox, we believe you have
+      the right to privacy on the internet. After all, the web is your home away from home, your workspace or your
+      storage unit.
+      {% endtrans %}
+    </p>
+    <img src="{{ static('img/firefox/features/safebrowser-lock.png')}}" alt="">
+    <p>
+      {% trans %}
+      When stepping inside a home or office, it's your right to be comfortable, safe and secure. There's no looking
+      over your shoulder. You can shut the blinds or choose not to answer the door. The web should be no different.
+      {% endtrans %}
+    </p>
+    <p>
+      {% trans %}
+      That is why we make Firefox with security features and add-ons that help us make the web safer for you. In this
+      tech
+      company, that means telling you what we know about you, and why we collect that information. Writing a privacy
+      policy
+      that's straightforward, and coming up with tools that protect your privacy.
+      {% endtrans %}
+    </p>
+    <p>
+      {% trans %}
+      There are lots of other ways that can make a safe browser. Below is a list of the most important ones we focus
+      on at
+      Firefox.
+      {% endtrans %}
+    </p>
+
+    <section class="article-section">
+      <h2>{{ _('Password Management') }}</h2>
+      <p>
+        {% trans %}
+        A password manager should help you create strong passwords and protect them across accounts. Most people can't
+        remember
+        every single password, especially if they're secure passwords. (Unless you have a photographic memory, in which
+        case,
+        congratulations!)
+        {% endtrans %}
+      </p>
+      <p>
+        {% trans %}
+        Firefox Password Manager stores all usernames and passwords in an encrypted format on your hard drive. We don't
```

Comment:
'Firefox Password Manager' links to https://support.mozilla.org/kb/password-manager-remember-delete-change-and-import

'Master Password' links to https://support.mozilla.org/kb/use-master-password-protect-stored-logins